broker-core/src/main/java/io/zeebe/broker/subscription/message/processor/OpenMessageStartEventSubscriptionProcessor.java

```diff
@@ -44,6 +48,10 @@ public void processRecord(
     final MessageStartEventSubscriptionRecord subscription = record.getValue();
     subscriptionState.put(subscription);
 
+    workflowState
+        .getEventScopeInstanceState()
+        .createIfNotExists(subscription.getWorkflowKey(), Collections.EMPTY_LIST);
```

Comment:
`Collections.emptyList()` is the type safe way of using `Collections.EMPTY_LIST`.